app/controllers/api/applications_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::ApplicationsController < ActionController::API
+  def index
+    render json: [
```

Comment:
Can we start out with returning an object that has a key `applications` that points to an array? Just so we're backwards compatible. Happy to also leave it out at this point and add it later. :)

Reply:
I actually started with this, then noticed [our API spec](https://editor.swagger.io/?url=https://raw.githubusercontent.com/DFE-Digital/apply-for-postgraduate-teacher-training-api/master/api_spec.yml) did it the other way! 

I think we should change both.

Reply:
Ah, yes, agreed!

Reply:
Actually, I say that, and we might have already put this spec in front of people, so changing it from under them might be worth thinking quickly about. 

@himal-mandalia - who have we sent [our swagger link](https://editor.swagger.io/?url=https://raw.githubusercontent.com/DFE-Digital/apply-for-postgraduate-teacher-training-api/master/api_spec.yml) to?

(we'll get it changed, it's just a question of whether _right now_ is correct)

Reply:
@kjdchapman we've sent this out to three vendors so we shouldn't be changing it yet - some of them may not have had a chance to look at it yet.

Reply:
Added this as [an issue on this repo](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/issues/37), and a [related one on the API spec repo](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-api/issues/3), so we don't lose sight of it.

Reply:
We might also want to look into HATEOS style payload with links.